joplin/base/models/__init__.py

```diff
@@ -68,6 +60,7 @@ def __str__(self):
         return self.title
 
 
+# I tried to remove the ProcessPages, but ended up getting errors with old migrations. - chia 1/22/20
```

Comment:
will this become a "comment with hours wasted trying to remove process pages" type thing? the world may never know...